packages/cli/src/commands/tablegen.ts

```diff
@@ -19,11 +19,11 @@ const commandModule: CommandModule<Options, Options> = {
   },
 
   async handler({ configPath }) {
-    const srcDirectory = await getSrcDirectory();
+    const codegenDir = await getCodegenDirectory();
 
     const config = await loadStoreConfig(configPath);
 
-    await tablegen(config, srcDirectory);
+    await tablegen(config, codegenDir);
```

Comment:
`src` doesn't need a config option because you can just change it in `foundry.toml`,
you can't change `codegen` though, would be nice to have it be configurable